ground/src/main/res/values/dimens.xml

```diff
@@ -39,4 +39,16 @@
   <dimen name="job_card_corner_radius">16dp</dimen>
   <dimen name="job_card_start_end_margin">8dp</dimen>
   <dimen name="zoom_on_cluster_click_padding">50dp</dimen>
+
+  <!-- Standard Spacing -->
+  <dimen name="spacing_small">4dp</dimen>
```

Comment:
CI is failing because these are unused. Maybe add the first reference to them to this PR and use elsewhere as needed in follow-ups?